release-team/role-handbooks/features/README.md

```diff
@@ -0,0 +1,139 @@
+  - Draft and / or review https://kubernetes.io/blog/ release announcement post, leveraging the themes collected across the release cycle e.g., [1.11 Announcement][1.11-announcement]
+  - Engage with media analysts during the embargo period to discuss the release themes
+  - CNCF Announcement webinar
+  - Identify potential contributors for the “5 Days of Kubernetes” blog series
+  - Identify candidates to assume the Features Lead role (according to the [Release Team selection process][rt-selection]) in the following release cycle
+
+## General Requirements
+
+In addition to the [requirements detailed for all Release Team members][rt-requirements], becoming a Features Lead has additional gates:
+  - MUST have served on the Release Team in a previous capacity, ideally as a Features Shadow
+  - MUST be a member of the SIG Release Google Group
+  - MUST be a member of the SIG PM Google Group
+  - MUST be a member of the Kubernetes Milestone Burndown Google Group
+
+Helpful characteristics of a Features Lead include:
+- experience with the Kubernetes community, code layout, ecosystem projects, organizational norms, governance, SIG structure, architecture, and release process
+- product / project / program management experience
+- release management experience
+
+## Mentoring Shadows
+
+## Getting Started
+
+### Access Required
+Ensure that the previous Features Lead has given (or facilitated getting) you access to:
+- GitHub teams
+  - features-maintainers
+  - kubernetes-milestone-maintainers
+- OWNERS_ALIASES (as `features-maintainers` in [kubernetes/features][k/features] repo)
+- Edit access to the Kubernetes x.y.0 Features Tracking Sheet e.g., [Kubernetes 1.12 Features Tracking][1.12-tracking]
+
+## Process
+
+### Standards
+As mentioned previously, the Features Lead role encompasses several cross-functional responsibilities with [SIG PM][sig-pm].
+
+The process of maintaining a feature in Kubernetes is documented in the [kubernetes/features][k/features] repo. Any questions / concerns / suggestions for improvement to the Features process should be raised as GitHub issues / PRs to k/features.
+
+It is important that this process be followed and documentation remain up-to-date as the [Features repo][k/features] is the primary ingress point for contributors interested in tracking features.
+
+### Milestone Activities + Timing
+- Link items from succession section
+- (optional) Watch the kubernetes/features repo to get notified on all issues
+- Ensure that Feature Tracking sheet for the new release was created by the previous Features Lead and populated with the features removed from the previous milestone
+- Features Tracking sheet is shortlinked with the pattern `k8sxyy-features` e.g., http://bit.ly/k8s112-features
+- Ensure the Features Tracking sheet has the appropriate permissions:
+ownership transferred to you
+  - anyone who has the link can comment
+  - Edit access
```

Comment:
Let's not publish the emails here explicitly (to lower the chance that they'll be spammed).